Makefile

```diff
@@ -54,7 +54,7 @@ BLE_ADDR ?= "none"
 # JerryScript options
 JERRY_BASE ?= $(ZJS_BASE)/deps/jerryscript
 EXT_JERRY_FLAGS ?=	-DENABLE_ALL_IN_ONE=ON \
-			-DFEATURE_PROFILE=$(ZJS_BASE)/jerry_feature.profile \
+			-DFEATURE_PROFILE=$(ZJS_BASE)/fragments/jerry_feature.profile \
```

Comment:
I'd like to not be leaving generated files randomly all around the tree. The best place for them is outdir/. Also maybe this should be board-specific like outdir/$BOARD?

If not in outdir than FWIW I think root is better than hidden in a subdir.